tests/test_stack.py

```diff
@@ -31,5 +31,45 @@ def test_called_by_current_file(self):
         file = Path(__file__)
         assert was_called_by(frame, file)
 
+    def test_called_by_symlink_file(self, tmp_path):
+        # Set up a call from a linked file in a temp directory,
+        # then test it with its resolved path.
+        # Here we expect `was_called_by` to recognize
+        # "`tmp_path`/foo_link.py" as same as "`tmp_path`/foo.py".
```

Comment:
```suggestion
        # "`tmp_path`/caller_link.py" as same as "`tmp_path`/caller.py".
```

A bit more speaking names will help future readors of the test files to graps the test more quickly :)